### PR TITLE
Update documentation about build process

### PR DESCRIPTION
**Pull request by katrialesser**
# READY FOR REVIEW
- Improve documentation, that's all.

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by? no rush

# Urgency
- No rush, low

# Steps to Test

1. Look at readme to see updated command for Development

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? NA

# Associated Issues and/or People
- JIRA ticket - NA
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by rebeccahongsf — APPROVED** [review]
This looks good to me! 😄  Feel free to merge away @katrialesser!

**Review by sherakama — APPROVED** [review]
